rust/src/client.rs

```diff
@@ -250,8 +250,8 @@ pub(crate) fn microdnf_install(args: Vec<String>) -> Result<()> {
 
     let microdnf_status = Command::new(microdnf_command)
         .arg("install")
-        .args(&args)
         .arg("-y")
+        .args(&args)
```

Comment:
This is a good change but it looks unrelated.  I'm fine adding it, but as a general rule let's keep commits logically separate.  See e.g. https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

I think a whole separate PR is overkill, but a separate commit in this PR would be a good best practice.